sparta/src/Destination.cpp

```diff
@@ -51,26 +51,29 @@ void DefaultFormatter::write(const sparta::log::Message& msg)
 {
     std::ios::fmtflags f = stream_.flags();
 
-    stream_ << '{';
+    if(msg.print_info)
```

Comment:
Users may want to output headerless message

Reply:
This kinda mimics the `sparta::log::RawFormatter`'s behavior...